ruby_on_rails/template.rb

```diff
@@ -30,3 +33,15 @@
 end
 
 run "chmod +x bin/fastcheck"
+
+after_bundle do
```

Comment:
I like the way you prompt the CLI users to understand how the CLI is setting up the project. Thanks!